README.md

```diff
@@ -113,7 +113,7 @@ Paths of included files can be either:
 - Local files:
   - Absolute paths (starting with a path separator).
   - Relative from the file that includes them (starting with `./` or `../`).
-  - Relative from the _docs/_ directory.
+  - Relative from the _docs/_ directory. For instance, _includes/header.md_ will match the file _docs/includes/header.md_.
```

Comment:
This is not necessarily true because you can change the root of the *docs/* directory using the [`docs_dir` setting](https://www.mkdocs.org/user-guide/configuration/#docs_dir). Please, remove it.

Reply:
what do you think of 

> Relative to your `docs_dir` directory. For instance if your `docs_dir` is _docs_, then _includes/header.md_ will match the file _docs/includes/header.md_` ?

Reply:
Yes, better than the current statement 👍🏼

Reply:
The french readme seems to be corrupted: https://github.com/mondeja/mkdocs-include-markdown-plugin/blob/master/locale/fr/README.md . Is the hook broken?

Reply:
Could not run the pre-commit hook on my M2 mac, could you check if everything s ok please ?

Reply:
Thanks for the contribution, done some format changes to stop breaking the translations and changed the wording in French and Spanish. I'll fix the hook ASAP.